examples/advanced/rooInterface/Makefile.am

```diff
@@ -13,13 +21,17 @@ input_workspaces = \
 	WS_Poisson.root \
 	WS_Poisson_withSystematics.root
 
+CLEANFILES += *.txt
 $(input_workspaces): WS_%.root: PrepareWorkspace_%.C
-	$(ROOTEXEC) -l -b -q $<
+	$(ROOTEXEC) -l -x -q $<
 
-TEST_EXTENSIONS += .root
 ROOT_LOG_COMPILER = $(builddir)/runRooInterface
 
-# Disabled:
-# TESTS = $(input_workspaces)
+TESTS = $(input_workspaces) runBATCalculator.C
+
+# only works with root 5
+if WITH_CLING
+XFAIL_TESTS = runBATCalculator.C
```

Comment:
> when it will pass, some day, because of other fixes

Ok, it won't accidentally work I guess :smile: 
I wonder why it fails, perhaps your test driver can't deal with XFAIL...
In any case, I will just deactivate the test for root6. Assuming that you still test  with root5, at least we can be sure it is tested before release even though users of root6 can't run the example. Oh well, let's wait if anyone complains.